### ISSUE TITLE
Static body linear_velocity as surface tangent speed

### ISSUE THREAD

**Issue by briansemrau** [briansemrau/godot_box2d#54]
In Godot you can set a linear velocity on static bodies so that they behave like conveyor belts.

This is probably the best way to implement it:
http://www.iforce2d.net/b2dtut/conveyor-belts

**Comment by jordo** [comment]
heh, I'm confused about this

**Comment by jordo** [comment]
linear velocity on static bodies?

**Comment by briansemrau** [comment]
Yeah, it doesn't move it but it changes collision behavior: https://docs.godotengine.org/en/latest/classes/class_staticbody2d.html#class-staticbody2d-property-constant-linear-velocity

**Comment by jordo** [comment]
Oh weird, ya, I guess godot doesn't expose any solver interaction, because this is trivial to do in pre-solve.
